customer-commons/src/main/java/no/unit/nva/customer/model/interfaces/Customer.java

```diff
@@ -0,0 +1,52 @@
+package no.unit.nva.customer.model.interfaces;
+
+import java.time.Instant;
+import java.util.UUID;
+
+public interface Customer {
```

Comment:
I believe it is fine this lives here instead of in the `nva-datamodel-java` , it belongs to the (micro)service.